src/js/components/streaming-api-handler.tsx

```diff
@@ -300,14 +300,15 @@ class StreamingAPIHandler extends React.Component<Props, void> {
     if (accessToken && teamId) {
       // Logged in, inside the app. Deployment View was not the landing page
       url = `${streamingAPIUrl}/${teamId}?token=${encodeURIComponent(accessToken)}`;
```

Comment:
Good point. Made the change. It added a bit of complexity to this component since I didn't want the UI to open the deployment-only streaming connection while the team information was being loaded.